### PR TITLE
Posts have own view

### PR DESCRIPTION
**Pull request by SaphMB**
Users can now click on the 'posted at' link and see the post within a single post view. Single post views have dynamic urls and the user can return to the posts view from a single post view.

Screenshots of post and single post views:
image

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -14,10 +14,14 @@ def index
     @posts = Post.reverse_order
   end
 
+	def show
```
Comment (dearshrewdwit) [inline app/controllers/posts_controller.rb]: indentation

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -9,9 +9,12 @@
       </span>
       <br>
     </div>
-    <div class="post__body">
-      <%= simple_format(post.message)%>
-    </div>
+		<a class="body_link" href="/posts/<%=post.id%>">
```
Comment (dearshrewdwit) [inline app/views/posts/index.html.erb]: indentation

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]

**Comment by artemisxen** [comment]
@dearshrewdwit conflicts resolved. Can you review please?

File: spec/features/user-can-view-post-at-its-dynamic-url.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.feature "Timeline", type: :feature do
+  let!(:post) { create(:post, message: "First post", :id 1) }
+
+  scenario "can view a single post" do
+    visit "/posts/1"
```
Comment (dearshrewdwit) [inline spec/features/user-can-view-post-at-its-dynamic-url.rb]: this should be a helper path method

Comment (dearshrewdwit) [reply spec/features/user-can-view-post-at-its-dynamic-url.rb]: `post_path(post)` something like

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -5,7 +5,7 @@
         <%= post.id  %>
       </span>
       <span class="post__head__date">
-        posted at: <%= post.posted_at %>
+	<a class="body_link" href="/posts/<%=post.id%>"> posted at: <%= post.posted_at %></a>
```
Comment (dearshrewdwit) [inline app/views/posts/index.html.erb]: this should be a helper method like `post_path(post)`

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]

**Comment by artemisxen** [comment]
@dearshrewdwit updated

**Comment by artemisxen** [comment]
@dearshrewdwit test passed

**Review by dearshrewdwit — DISMISSED** [review]

**Pull request closed without merging**